docs/README.md

```diff
@@ -1,43 +1,78 @@
+- **[Reference Guides](https://developer.wordpress.org/block-editor/reference-guides/)** - this section is the heart of the handbook and is where you can get down to the nitty-gritty and look up the details of the particular API that you’re working with or need information on. Among other things, the [Block API Reference](https://developer.wordpress.org/block-editor/reference-guides/block-api/) covers most of what you will want to do with a block, and each [component](https://developer.wordpress.org/block-editor/reference-guides/components/) and [package](https://developer.wordpress.org/block-editor/reference-guides/packages/) is also documented here. For components there is also the [Storybook tool](https://wordpress.github.io/gutenberg/?path=/story/docs-introduction--page).
+- **[Explanations](https://developer.wordpress.org/block-editor/explanations/)** - this section enables you to go deeper and reinforce your practical knowledge with a theoretical understanding of the [Architecture](https://developer.wordpress.org/block-editor/explanations/architecture/) of the block editor. There’s also a useful [Glossary of terms](https://developer.wordpress.org/block-editor/explanations/glossary/), and the [FAQs](https://developer.wordpress.org/block-editor/explanations/faq/) should answer any outstanding questions you may have.
+- **[Contributor Guide](https://developer.wordpress.org/block-editor/contributors/)e** - Gutenberg is open source software and anyone is welcome to contribute to the project. This section details how to contribute and can help you choose in which way you want to contribute, whether that be with [code](https://developer.wordpress.org/block-editor/contributors/code/), with [design](https://developer.wordpress.org/block-editor/contributors/design/), with [documentation](https://developer.wordpress.org/block-editor/contributors/documentation/), or in some other way.
 
-### Create a Block Tutorial
+### Further resources
+This handbook should be considered the canonical resource for all things related to block development. However there are other resources that can help you.
+#### Blog
+The [WordPress Developer Blog](https://developer.wordpress.org/news/) is an ever-growing resource of technical articles covering specific topics related to block development and covering a wide variety of use cases. The blog is also an excellent way to keep up with the latest developments in WordPress.
+#### Courses
```

Comment:
I would try to reduce the `Further Resources` section, which is nearly as long as the main one. For example, the `Courses` subsection is too detailed, considering we are on the Handbook landing page.